WooCommerce/UITestsFoundation/Screens/Orders/AddCustomAmountScreen.swift

```diff
@@ -0,0 +1,36 @@
+import ScreenObject
+import XCTest
+
+public final class AddCustomAmountScreen: ScreenObject {
+
```

Comment:
Super not a blocker, we don't leave an empty line after the class declaration

Reply:
I removed the extra line in e996e04fdaf224e5bac4ef6d096c551ca6d808e7, this is a valid point. Thanks @toupper! Once the CI is 🟢 again, I'm going for the :shipit:.